nwb_conversion_tools/basedatainterface.py

```diff
@@ -21,8 +22,14 @@ def get_metadata(self):
 
     @classmethod
     def get_conversion_options_schema(cls):
-        return get_schema_from_method_signature(cls.convert_data, exclude=['nwbfile', 'metadata_dict'])
+        return get_schema_from_method_signature(cls.convert_data, exclude=['nwbfile', 'metadata'])
 
     @abstractmethod
-    def convert_data(self, nwbfile_path, metadata_dict):
+    def run_conversion(self, nwbfile_path, metadata):
         pass
+
+    @abstractmethod
+    def convert_data(self, nwbfile_path, metadata):
+        warnings.warn("The convert_data method should now be renamed to run_conversion "
+                      "as of nwb-conversion-tools v0.6.0", DeprecationWarning)
```

Comment:
@bendichter Should this be a warning or a hard assertion on the deprication?

Reply:
This is good